win32/src/winapi/types.rs

```diff
@@ -85,7 +85,22 @@ pub fn io_error_to_win32(err: &std::io::Error) -> u32 {
 pub const MAX_PATH: usize = 260;
 
 #[inline]
-pub fn unix_nanos_to_filetime(nanos: u64) -> u64 {
+pub fn unix_nanos_to_filetime(nanos: i64) -> u64 {
```

Comment:
Is there a reason to pass around u64s here rather than FILETIMEs?  Feels like the time APIs were designed around not needing to pass around plain numeric types with functions that are careful about conversions...

Reply:
Originally I just liked using an actual u64 as opposed to FILETIME with 2 u32s. But given this lives in winapi, it probably makes sense to just accept/return FILETIME.

Reply:
I moved these to associated functions on FILETIME instead. I also updated the host interface to use Unix nanos instead of Windows's weird hecto-nanoseconds. Conversions now happen solely in win32.